webxr-api/hittest.rs

```diff
@@ -43,6 +45,19 @@ pub struct EntityTypes {
     pub mesh: bool,
 }
 
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "ipc", derive(serde::Serialize, serde::Deserialize))]
+/// Vec<EntityType>, but better
```

Comment:
orly?

Reply:
it's a vec in the spec, but there are only three possible values, and it's easier to store and iterate in this form

Reply:
Ok, but look at the lines following the line that I commented on. I think you copied a comment from another file.